Gruntfile.js

```diff
@@ -58,12 +58,14 @@ module.exports = function(grunt) {
 							slowTag = '--grep @slow';
 						} else if (tag === 'unstable') {
 							slowTag = '--grep @unstable';
+						} else if (tag === 'syncing') {
+							slowTag = '--grep @syncing';
```

Comment:
What is that?

Reply:
There was an use case to run tests when nodes were only broadcasting (not syncing). This gives us the possibility to run only those tests.

Reply:
You mean that @syncing means sync-only mode?

Reply:
Forget it, the test wasn't giving any value at the current format of small network with 10 peers. I've removed the possibility to narrow the scope of the PR. We can easily return it later.